src/main/scala/sparkz/core/network/NetworkController.scala

```diff
@@ -195,8 +191,18 @@ class NetworkController(settings: NetworkSettings,
       connectionToPeer(activeConnections, unconfirmedConnections)
   }
 
+  private def handleNewConnectionCreation(remoteAddress: InetSocketAddress, localAddress: InetSocketAddress): Unit = {
+    val connectionDirection: ConnectionDirection =
+      if (unconfirmedConnections.contains(remoteAddress)) Outgoing else Incoming
+    val connectionId = ConnectionId(remoteAddress, localAddress, connectionDirection)
+    log.info(s"Unconfirmed connection: ($remoteAddress, $localAddress) => $connectionId")
+    if (connectionDirection.isOutgoing && canEstablishNewOutgoingConnection) createPeerConnectionHandler(connectionId, sender())
+    else if (connectionDirection.isIncoming && canEstablishNewIncomingConnection) peerManagerRef ! ConfirmConnection(connectionId, sender())
+    else self ! ConnectionDenied(connectionId, sender())
+  }
+
```

Comment:
I know it was like this before, but can we please refactor this method a bit, cause it's very hard to read. Just some spaces here and there will help a lot, like
```
  private def handleNewConnectionCreation(remoteAddress: InetSocketAddress, localAddress: InetSocketAddress): Unit = {
    val direction = if (unconfirmedConnections.contains(remoteAddress)) Outgoing else Incoming
    val connectionId = ConnectionId(remoteAddress, localAddress, direction)
    log.info(s"Unconfirmed connection: ($remoteAddress, $localAddress) => $connectionId")
    
    if (direction.isOutgoing && canEstablishNewOutgoingConnection) 
      createPeerConnectionHandler(connectionId, sender())
    else if (direction.isIncoming && canEstablishNewIncomingConnection) 
      peerManagerRef ! ConfirmConnection(connectionId, sender())
    else 
      self ! ConnectionDenied(connectionId, sender())
  }
```

Reply:
yes, good point, thanks :)